### PR TITLE
Import multiple files

### PR DESCRIPTION
**Pull request by junajan**
Add possibility to import archived product files.

For example, from full export we get an archive with products saved in different files based on their product type. With this feature, we can take this archive and import it back to sphere.

### CODE REVIEW

File: src/coffee/import.coffee [hunk 1]
```diff
@@ -94,6 +103,28 @@ class Import
...
+      console.log "Loading files from", tempDir.name
+      filePaths = walkSync tempDir.name, { globs: ['**/*.csv'] }
+      if not filePaths.length
+        return Promise.reject "There are no CSV files in archive"
+
+      Promise.map filePaths, (fileName) =>
+        console.log "Processing file %s", fileName
+        filePath = path.join tempDir.name, fileName
+
+        @validator = new Validator(@options)
```
Comment (PhilippSpo) [inline src/coffee/import.coffee]: do we really need a new validator per file?

File: src/coffee/import.coffee [hunk 2]
```diff
@@ -94,6 +103,28 @@ class Import
...
+        fs.readFileAsync(filePath, {encoding: "UTF-8"})
+        .then (content) => @import content
+      , {concurrency: 1}
+      .then (res) ->
+        Promise.resolve([].concat.apply([], res))
```
Comment (PhilippSpo) [inline src/coffee/import.coffee]: what is this line doing?

Comment (junajan) [inline src/coffee/import.coffee]: it concats subarrays (from promise.map) to one array with all results

File: src/coffee/import.coffee (hunk 1 above)
Comment (junajan) [inline src/coffee/import.coffee]: I tried it without re-initing validator and it was failing inside of validator.

**Comment by junajan** [comment]
@emmenko when we do a full export of all products, products with different productTypes may have different attributes and CSV columns so we save them in different files based on their productType. Because we can export multiple files it should be also possible to import them back.

**Comment by emmenko** [comment]
@junajan ok, thanks for the explanation

**Pull request merged**